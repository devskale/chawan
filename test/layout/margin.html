<!doctype html>
<style>
.claz {
	margin: 1em 2em 1em;
}
.one {
	margin: 0 auto;
        width: 10em;
}
#a {
	margin: 1em;
}
</style>
<div style="width: 1000px">
spacer
<div class=claz>left, right: 2em<br>top, bottom: 1em.</div>
spacer
<div class="clazz one">width: 10em<br>left, right: auto<br>top, bottom: 0</div>
</div>
