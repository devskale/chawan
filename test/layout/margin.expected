spacer

    left, right: 2em
    top, bottom: 1em.

spacer
                                             width: 10em
                                             left, right: auto
                                             top, bottom: 0
